fixtures/demo_data.json

```diff
@@ -191,8 +191,8 @@
     "thumbnail": "demo/experiments_experiment/a/8/a8182b008f6434ba702ab177232625e2_thumbnail_1449675941_0564.png",
     "created": "2015-12-09T15:45:42.395Z",
     "slug": "universal-search",
-    "xpi_url": "https://people.mozilla.org/~jhirsch/universal-search-addon/addon.xpi",
-    "contribute_url": "https://github.com/mozilla/universal-search-addon/blob/master/CONTRIBUTING.md"
+    "xpi_url": "https://s3-us-west-2.amazonaws.com/universal-search/universal-search-1.0.0b1.xpi",
```

Comment:
Why a specific filename vs always just renaming to "universal-search.xpi" or "latest.xpi"?